### ISSUE TITLE
shortcut.agency

### ISSUE THREAD

**Issue by bestplayerbot** [DandelionSprout/adfilt#785]
## Which entry/entries are you submitting?

shortcut.agency
## Which things do they block, hide, or unbreak?

![image](https://user-images.githubusercontent.com/96731476/219823685-775a4c64-a86c-4d7a-abd2-e9942770b28a.png)
It doesn't need to be unblocked because the site is parked.
## Which of my lists are you submitting it to?
Anti-Malware
## Which adblocker(s) and version did you use when writing and testing the entries?

- [x] uBlock Origin
- [ ] AdGuard (Paid desktop version)
- [ ] AdGuard (Gratis browser versions that aren't Manifest V3 nor Safari)
- [ ] AdBlock (Firefox or Manifest V2)
- [ ] Adblock Plus (Firefox or Manifest V2)
- [ ] AdGuard Home
- [ ] Blokada
- [ ] I Don't Care About Cookies (The extension)
- [ ] AdNauseam

Other(s):

Adblocker version(s):
1.47.1b0
## Which filterlists did you use? Failing to tell this will temporarily close the report until it has been told.

Anti-Malware

**Comment by bestplayerbot** [comment]
`baam.agency` and `uphotel.agency` should  be allowed

**Comment by DandelionSprout** [comment]
Both those two seem to be perfectly legitimate long-term UK event companies, so I'll attempt to get them whitelisted now.
